src/settings/LocationLocations/RemoteStorage/Control.js

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { Select, Tooltip } from '@folio/stripes/components';
 
 import PropTypes from 'prop-types';
-import childrenOf from '@folio/stripes-components/util/childrenOf';
 import { FormattedMessage } from 'react-intl';
+
+import { childrenOf } from '../../../util/childrenOf';
```

Comment:
you should export from stripes-components, instead of code duplication

Reply:
We can't import it from stripes-components in the correct way, since it is imported from stripes-util. This story wants us to use escape private paths.

Reply:
again, you should export this util from library so it can be properly imported, what you do now is incorrect

Reply:
Done